cloudformation/tasking-manager.template.js

```diff
@@ -0,0 +1,344 @@
+const cf = require('@mapbox/cloudfriend');
```

Comment:
Is it possible to move this file and directory into our `devops` directory?

Reply:
yep! 👍